src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs

```diff
@@ -69,6 +69,21 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				SetupLayer();
 			else if (e.PropertyName == Controls.Frame.HasShadowProperty.PropertyName)
 				UpdateShadow();
+			else if (e.PropertyName == Microsoft.Maui.Controls.Frame.ContentProperty.PropertyName)
```

Comment:
Not related with your changes, but ideally, this Renderer should use the Mapper on iOS already happen on Android: https://github.com/dotnet/maui/blob/d749dc779531dcf491746fd2f190382db4f18e0d/src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs#L28

Reply:
I actually wanted to do it in the first place, but I wasn't sure if I should have 😅